internal/pkg/deploy/cloudformation/stack/testdata/workloads/job-manifest.yml

```diff
@@ -41,6 +43,13 @@ sidecars:
         path: '/var/www'
     variables:
       NGINX_PORT: 8080
+
+environments:
+  test:
+    image:
+      labels:
+        com.amazonaws.ecs.copilot.coollabel: Synecdoche
```

Comment:
Do you know if these overrides work for the `image` field? I thought we had to implement it for some reason

Reply:
Image field gets overridden but docker build happens before applying overrides.